app/experimenter/experiments/models/nimbus.py

```diff
@@ -300,6 +300,18 @@ def to_timestamp(date):
             slug=self.slug, from_date=start_date, to_date=end_date
         )
 
+    @property
+    def review_url(self):
+        is_staging = "settings-writer.stage" in settings.KINTO_ADMIN_URL
+
+        return "{base_url}/{collection_path}/{collection}/{review_path}".format(
+            base_url=settings.KINTO_ADMIN_URL,
+            collection_path="#/buckets/main-workspace/collections",
+            collection=self.application_config.rs_experiments_collection,
+            # TODO: Update when simple-review is on prod in remote settings
```

Comment:
I'd file this as an actual ticket and put the issue ID right here in the comment